### ISSUE TITLE
Visual focus indicator is not visible properly under “Edit Button” pop up controls.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#115]
User Experience:

If, the visual focus indicator is not visible on the checkbox, then the user who depends upon keyboard for navigation might get affected hence failed to identify the current focus on the control.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open published Developer portal (in incognito mode) at https://accessibility.developer.azure-api.net/.

2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Button and select it.
8. Navigate to the "Button" dialog and select edit button.
9. Navigate to the various controls of the "Button" dialog box like label, link to, appended, size etc.
10. Verify, whether the visual focus indicator is visible on controls or not.

 

Actual Result:

Visual focus indicator is not visible properly under “Edit Button” pop up controls like like label, link to, appended, size etc.

Expected Result:

Visual focus indicator should be visible properly under “Edit Button” pop up. Focus indicator should be in square dotted form.

 

Note:

This issue does exist on the chrome browser.

         

MAS Reference:

MAS 2.4.7 - Focus Visible

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
